File: benchmarks/javascript/sort_numbers.js
const n = Number(process.argv[2] ?? 300000);
const numbers = [];
let seed = 1;
for (let i = 0; i < n; i++) {
    seed = (seed * 75 + 74) % 65537;
    numbers.push(seed);
}
numbers.sort((a, b) => a - b);
console.log(numbers[0], numbers[Math.floor(n / 2)], numbers[n - 1]);

File: benchmarks/lua/sort_numbers.lua
local n = tonumber(arg[1] or "300000")
local numbers = {}
local seed = 1
for i = 0, n - 1 do
    seed = (seed * 75 + 74) % 65537
    numbers[#numbers + 1] = seed
end
table.sort(numbers)
print(numbers[1] .. " " .. numbers[n // 2 + 1] .. " " .. numbers[n])
